package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "nanoexperiencias-digitalbank",
+  "version": "1.0.0",
+  "description": "automatizacion de pagina",
+  "main": "index.js",
+  "scripts": {
+    "test": "wdio ./wdio.conf.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/luciagonzalezdm/nanoexp-digitalbank.git"
+  },
+  "author": "luciagonzalez",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/luciagonzalezdm/nanoexp-digitalbank/issues"
+  },
+  "homepage": "https://github.com/luciagonzalezdm/nanoexp-digitalbank#readme",
+  "devDependencies": {
+    "webdriverio": "^7.16.12"
```

Comment:
This project needs more libraries. Consider installing mocha, chai and chai-webdriverio (and every library that wdio initially installs when you are configuring the project) again, so you `package.json` reflects them (as this is what a user of your repo will need to reproduce your environment)